src/pages/patientView/sampleHeader/SampleSummaryList.tsx

```diff
@@ -0,0 +1,182 @@
+import * as React from 'react';
+import SampleManager from '../SampleManager';
+import { ClinicalDataBySampleId } from 'cbioportal-ts-api-client';
+import _ from 'lodash';
+import { getSpanElementsFromCleanData } from '../clinicalInformation/lib/clinicalAttributesUtil';
+import { getMouseIcon } from '../SVGIcons';
+import { getSampleViewUrl } from 'shared/api/urls';
+import SignificantMutationalSignatures from '../patientHeader/SignificantMutationalSignatures';
+import { PatientViewPageStore } from '../clinicalInformation/PatientViewPageStore';
+import { If, Then, Else } from 'react-if';
+import LoadingIndicator from 'shared/components/loadingIndicator/LoadingIndicator';
+
+export type ISampleSummaryListProps = {
+    sampleManager: SampleManager;
+    patientViewPageStore: PatientViewPageStore;
+    handleSampleClick: (
+        id: string,
+        e: React.MouseEvent<HTMLAnchorElement>
+    ) => void;
+    toggleGenePanelModal: (genePanelId?: string) => void;
+    genePanelModal: any;
+    handlePatientClick: (id: string) => void;
+};
+export default class SampleSummaryList extends React.Component<
+    ISampleSummaryListProps,
+    {}
+> {
+    public render() {
+        let sampleHeader: (JSX.Element | undefined)[] | null = null;
+        sampleHeader = _.map(
+            this.props.sampleManager.samples,
+            (sample: ClinicalDataBySampleId) => {
+                if (
+                    !this.props.sampleManager.isSampleVisibleInHeader(sample.id)
+                ) {
+                    return undefined;
+                }
+
+                const isPDX: boolean =
```

Comment:
lets put this into a method of the class (isPDX)

Reply:
Sure, Hongxin just merged a pr that also added content into summary list, I will address them together.